vpn_slice/dnspython.py

```diff
@@ -22,6 +22,7 @@ def lookup_host(self, hostname, keep_going=True):
         result = set()
 
         for source in self.bind_addresses or [None]:
+            source = ip_interface(source)
```

Comment:
That's good thinking. I took it one step further, de-interfacing the input in the base `DNSProvider` so that we don't run into any problems for folks still using the `DigProvider`.